openrr-planner/src/collision/self_collision_checker.rs

```diff
@@ -0,0 +1,150 @@
+use std::{path::Path, sync::Arc, time::Duration};
+
+use arci::TrajectoryPoint;
```

Comment:
Please do not depend on `arci`. Use TrajectoryPoint in funcs.rs instead. (It might be better to move from funcs.rs to something.rs)
(In the future I might change my mind, but currently, I think that openrr-planner should independent from `arci`)